docs/source/contributing/developer_guide.md

```diff
@@ -542,10 +542,10 @@ There are some example in the ``CompoundStep`` doc:
 The base class for most MCMC sampler (except SMC) is in [ArrayStep](https://github.com/pymc-devs/pymc/blob/main/pymc/step_methods/arraystep.py).
 You can see that the ``step.step()`` is mapping the ``point`` into an array, and call ``self.astep()``, which is an array in, array out function.
 A PyMC model compiles a conditional logp/dlogp function that replace the input RVs with a shared 1D tensor (flatten and stack view of the original RVs).
-And the transition kernel (i.e., ``.astep()``) takes an array as input and output an array.
-See for example in the [MH sampler](https://github.com/pymc-devs/pymc/blob/6d07591962a6c135640a3c31903eba66b34e71d8/pymc/step_methods/metropolis.py#L139-L173).
+And the transition kernel (i.e., ``.astep()``) takes an array as input and outputs an array.
+See for example in the [MH sampler](https://github.com/pymc-devs/pymc/blob/89f6fcf751774fb50016561dc448a87fba7ed3aa/pymc/step_methods/metropolis.py#L235-L289).
```

Comment:
```suggestion
For example, see the [MH sampler](https://github.com/pymc-devs/pymc/blob/89f6fcf751774fb50016561dc448a87fba7ed3aa/pymc/step_methods/metropolis.py#L235-L289).
```
Nitpick: perhaps this reads a bit better?

Reply:
Sure! 😄